lang/cs/Org.Apache.REEF.Wake/Avro/ProtocolSerializer.cs

```diff
@@ -36,36 +36,52 @@ public sealed class ProtocolSerializer
     {
         private static readonly Logger Logr = Logger.GetLogger(typeof(ProtocolSerializer));
 
-        // Delagates for message serializers and deserializers.
+        /// <summary>
+        /// Delegate for message serializer.
+        /// </summary>
         private delegate void Serialize(MemoryStream stream, object message);
+
```

Comment:
Yes, we use Doxygen and post API docs on [reef.apache.org](http://reef.apache.org). Not sure if we publish docs on private members, though - but in general we do use docstrings to document them.